### DISCUSSION TITLE
Managing bindings based on active TabPane

### DISCUSSION THREAD

**Discussion by ghost** [Textualize/textual#5102] [Q&A] [answered]
How can I manage bindings based on the TabPane that is currently active?

I would like to see bindings like ("a", "add_project", "Add project") when switching to a TabPane with id "projects", and bindings like ("a", "add_task", "Add task") when switching to a TabPane with id "tasks". I managed to make it so that different bindings are displayed in the footer, but the actions do not execute.

Thanks :)

**Comment by willmcgugan** [comment, marked as answer]
If your bindings display in the footer, but don't run, it likely means the action syntax is wrong.  Where is `action_add_project` defined? If it is not in your TabPane class, you will need a namespace prefix. i.e. `app.add_project`, if the action is defined on the app. Or `screen.add_project` if it is defined on the screen.

**Reply by ghost** [reply]
I was able to achieve the expected result with your help, thank you :)
